awx/ui/src/components/JobList/JobListItem.js

```diff
@@ -167,6 +168,17 @@ function JobListItem({
                   />
                 )}
               <LaunchedByDetail job={job} />
+              {job.launch_type === 'scheduled' && (
+                <Detail
+                  dataCy="job-schedule"
+                  label={t`Schedule`}
+                  value={
+                    <Link to={getScheduleUrl(job)}>
+                      {job.summary_fields.schedule.name}
```

Comment:
You could add `schedule` to the destructed job object [here](https://github.com/ansible/awx/pull/11792/files#diff-04d85be33263d77d4c6a8ba191660b3524a19d0b96d788ded224e23484693551R45-R54)